src/mame/machine/eispc_kb.cpp

```diff
@@ -330,6 +338,8 @@ void eispc_keyboard_device::device_add_mconfig(machine_config &config)
 	m_mcu->out_p1_cb().set([this](uint8_t data)
 	{
 		LOGPORTS("Writing %02x PORT 1\n", data);
+		LOGLEDS("Num: %d\n", BIT(data, 7));
+		m_led_num_cb(BIT(data, 7) ? 1 : 0);
```

Comment:
The `? 1 : 0` is redundant here – `BIT` returns the same type as its first argument, not `bool`, so in this case it will return `uint8_t` (it's not going to break anything, it's just extra clutter).

Reply:
Right, I wasn't just sure if the value would be 0x80 or 0x01 for the callback so I made sure it was 0x01 if the bit was set. On the other hand if I'd been thinking about the LOG printout I would have known, I clean it up next commit.